ERCS/erc-7656.md

```diff
@@ -0,0 +1,279 @@
+
+  /**
+   * @notice Creates a token linked account for a non-fungible token.
+   * If account has already been created, returns the account address without calling create2.
+   * @param implementation The address of the implementation contract
+   * @param salt The salt to use for the create2 operation
+   * @param chainId The chain id of the chain where the account is being created
+   * @param tokenContract The address of the token contract
+   * @param tokenId The id of the token
+   * Emits Created event.
+   * @return account The address of the token linked account
+   */
+  function create(
+    address implementation,
+    bytes32 salt,
+    uint256 chainId,
+    address tokenContract,
+    uint256 tokenId
+  ) external returns (address account);
+
+  /**
+   * @notice Returns the computed token linked account address for a non-fungible token.
+   * @param implementation The address of the implementation contract
+   * @param salt The salt to use for the create2 operation
+   * @param chainId The chain id of the chain where the account is being created
+   * @param tokenContract The address of the token contract
+   * @param tokenId The id of the token
+   * @return account The address of the token linked account
+   */
+  function compute(
+    address implementation,
+    bytes32 salt,
+    uint256 chainId,
+    address tokenContract,
+    uint256 tokenId
+  ) external view returns (address account);
+}
+```
+
+Any contract created using a `ERC7656Registry` SHOULD implement the `IERC7656Contract` interface:
+
+```solidity
+interface IERC7656Contract {
+  /**
+  * @notice Returns the token linked to the contract
+  * @return chainId The chainId of the token
+  * @return tokenContract The address of the token contract
+  * @return tokenId The tokenId of the token
+  */
+  function token() external view returns (uint256 chainId, address tokenContract, uint256 tokenId);
+
+}
+```
+
+or the `IERC6551Account` interface
+
+```solidity
+/// @dev the ERC-165 identifier for this interface is `0x6faff5f1`
+interface IERC6551Account {
+    receive() external payable;
+
+    function token()
+        external
+        view
+        returns (uint256 chainId, address tokenContract, uint256 tokenId);
+
+    function state() external view returns (uint256);
+
+    function isValidSigner(address signer, bytes calldata context)
+        external
+        view
+        returns (bytes4 magicValue);
+}
+```
+
+or both. This flexibility makes [ERC-6551](./eip-6551.md) accounts compatible with this proposal out-of-the-box.
+
+_Notice that most likely the function state will return a bytes32 value in the final version of [ERC-6551](./eip-6551.md). As soon as that changes, we will update this proposal accordingly._
+
+The reference implementation of `ERC7656Registry` is mutated from `ERC6551Registry`, with minor changes to emit a different event and error, and a `supportsInterface` function that returns true only for the `IERC7656Registry` interfaceId.
```

Comment:
```suggestion

## Reference Implementation 

The reference implementation of `ERC7656Registry` is mutated from `ERC6551Registry`, with minor changes to emit a different event and error, and a `supportsInterface` function that returns true only for the `IERC7656Registry` interfaceId.
```

Should be a separate section here.

Reply:
It was. By mistake I removed the paragraph title